halyard-web/src/main/java/com/netflix/spinnaker/halyard/controllers/v1/PersistentStorageController.java

```diff
@@ -46,8 +47,8 @@
 
   @RequestMapping(value = "/", method = RequestMethod.GET)
   DaemonTask<Halconfig, PersistentStorage> getPersistentStorage(@PathVariable String deploymentName,
-      @RequestParam(required = false, defaultValue = DefaultControllerValues.validate) boolean validate,
-      @RequestParam(required = false, defaultValue = DefaultControllerValues.severity) Severity severity) {
+                                                                @RequestParam(required = false, defaultValue = DefaultControllerValues.validate) boolean validate,
```

Comment:
Small nit, could you preserve the existing indentation for multiline parameter definitions?